thumbor/engines/pil.py

```diff
@@ -452,46 +451,22 @@ def read(  # noqa
         return results
 
     def read_multiple(self, images, extension=None):
-        gif_writer = GifWriter()
-        img_buffer = BytesIO()
-
-        duration = []
-        converted_images = []
-        coordinates = []
-        dispose = []
-
-        for image in images:
-            duration.append(image.info.get("duration", 80) / 1000)
-            converted_images.append(image.convert("RGB"))
-            coordinates.append((0, 0))
-            dispose.append(1)
-
-        loop = int(self.image.info.get("loop", 1))
-
-        images = gif_writer.convertImagesToPIL(converted_images, False, None)
-        gif_writer.writeGifToFile(
-            img_buffer, images, duration, loop, coordinates, dispose
+        _, tmp_file_path = mkstemp()
```

Comment:
@scorphus, you can also create the temp file using a context manager: https://docs.python.org/3/library/tempfile.html#examples

Reply:
Seems like I overlooked that one. Thanks, @guilhermef, good catch!